sources/updater/waves/ohno.toml

```diff
@@ -0,0 +1,16 @@
+# NOTE: Update waves have an implicit start time. The initial wave starts
+# "now" or at the time of deployment and includes all seeds up to but not
+# including the `end_seed`. The first wave ends and the second wave begins
+# starting at the `end_time` of the first wave. The second wave includes
+# all seeds from the first two waves, and so on and so forth.
+
+# The following represents an "emergency" set of update waves for a rapid
+# deployment. The deployment lasts for 3 hours, with a small initial wave,
+# and then all nodes will be updated after the first hour.
+[[wave]]
+end_time = 'in 1 hour'
+end_seed = 64
+
+[[wave]]
+end_time = 'in 3 hours'
```

Comment:
I think this emphasizes how weird the end time of the last wave is.  It never really ends, it's just superceded.  Can we make it an optional field?

Or switch to start time?  I think that makes more sense anyway... you could delay the first wave's start that way, but you can't with this way.

Reply:
See Sam's reply: https://github.com/bottlerocket-os/bottlerocket/pull/883#discussion_r400374952

The "start time" of these waves is implicit. `end_time` or `start_time` are really just words to (hopefully) describe this key's purpose. This key constitutes the end of the wave for all seeds up to the current seed and the start of the next seed's wave.

So depending on how you think about it, it could be either.

I can see it either way and as long as we get consensus and document it I'm happy.

Reply:
paging @bcressey , @etungsten , @sam-aws